dolphinscheduler-alert/dolphinscheduler-alert-server/src/main/java/org/apache/dolphinscheduler/alert/AlertSenderService.java

```diff
@@ -93,6 +97,13 @@ public void send(List<Alert> alerts) {
                 continue;
             }
             AlertData alertData = new AlertData();
+            if (Objects.nonNull(alert.getContent())
+                    && !alert.getContent().startsWith("[")
```

Comment:
It's better to check by `jsonNodes instanceof JSONArray`. WDYT?

Reply:
Since jsonNodes are not related to JSONArray, instanceof cannot be used. I also tried isArray(), but I found that isArray() always returns false

Reply:
> It's better to check by `jsonNodes instanceof JSONArray`. WDYT?

Since jsonNodes are not related to JSONArray, instanceof cannot be used. I also tried isArray(), but I found that isArray() always returns false